publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -231,8 +244,12 @@ public void createRabbitMqConnection() {
             for (int i = 0; i < channelsCount; i++) {
                 rabbitChannels.add(rabbitConnection.createChannel());
             }
+            for (int i = 0; i < rabbitChannels.size(); i++){
```

Comment:
Is there a need to have two loops, i.e. line 244 and 247? It's easier to understand what's going on when done in in one loop:

for (int i = 0; i < channelsCount; i++) {
    ... channel = rabbitConnection.createChannel();
    channel.confirmSelect();
    rabbitChannels.add(channel);
}

Reply:
will change it